.travis.yml

```diff
@@ -55,15 +55,8 @@ jobs:
       env: DB=sqlite DEPENDENCIES=low
       install: travis_retry composer update --prefer-dist --prefer-lowest
 
-    - stage: Lint
-      env: DB=none
-      before_script:
-        - echo "extension=redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-        - travis_retry composer require --dev --prefer-dist --prefer-stable phpstan/phpstan:^0.8 symfony/console:^3.0
-      script: vendor/bin/phpstan analyse -l 1 -c phpstan.neon lib
-
-    - stage: Coverage
-      env: DB=sqlite
+    - stage: Test
+      env: DB=sqlite COVERAGE
```

Comment:
Why did we move coverage to tests stage? It slows down whole stage now from 2-3min to 7min before CQ is started.